tailwind.config.js

```diff
@@ -3,7 +3,36 @@ module.exports = {
     './pages/**/*.{js,ts,jsx,tsx}',
     './components/**/*.{js,ts,jsx,tsx}'
   ],
-  theme: {},
+  theme: {
+    colors: {
+      transparent: 'transparent',
+      current: 'currentColor',
+      gray: {
+        0o0: '#ffffff',
+        0o5: '#f5f6f7',
+        10: '#dfdfe2',
+        15: '#d0d0d5',
+
+        // DARK
+        75: '#3b3b4f',
+        80: '#2a2a40',
+        85: '#1b1b32',
+        90: '#0a0a23'
+      },
+      primary: {
```

Comment:
My preference would be to prefix with FCC. What do you think @GuillermoFloresV @utsab 

ex:
fcc-gray-90
fcc-primary-purple

Reply:
We can discuss and proceed it that way.

Reply:
Yeah, I think this works well. That way new contributors know that the color's name came directly from FCC.